### PR TITLE
Feature/jjdd8po 36 filtrowanie

### PR DESCRIPTION
**Pull request by michal-brakon**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: web/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <data-source>
         <name>java:ds/libraryDS</name>
         <class-name>com.mysql.cj.jdbc.MysqlDataSource</class-name>
-        <url>jdbc:mysql://mysql/library?allowPublicKeyRetrieval=true&amp;useSSL=false&amp;useLegacyDatetimeCode=false&amp;serverTimezone=Europe/Warsaw</url>
+        <url>jdbc:mysql://localhost:3306/library?allowPublicKeyRetrieval=true&amp;useSSL=false&amp;useLegacyDatetimeCode=false&amp;serverTimezone=Europe/Warsaw</url>
         <user>root</user>
```
Comment (DariuszCzajkowski) [inline web/src/main/webapp/WEB-INF/web.xml]: trzeba zmienić bo się nie zbuduje

File: web/src/main/java/com/infoshareacademy/domain/entity/LiteratureKind.java [hunk 2]
```diff
@@ -18,6 +21,9 @@
     @Column(name = "name")
     private String name;
 
+    @OneToMany(mappedBy = "kind")
+    private List<Book> books = new ArrayList<>();
```
Comment (DariuszCzajkowski) [inline web/src/main/java/com/infoshareacademy/domain/entity/LiteratureKind.java]: tutaj jeszcze fetch type lazy

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**